### PR TITLE
Catch errors with no matching preKey

### PR DESCRIPTION
**Pull request by neekolas**
## Summary

Helps mitigate INC-2, the No Matching PreKey error incident

### CODE REVIEW

File: src/Client.ts [hunk 1]
```diff
@@ -369,7 +369,7 @@ export default class Client {
         const res = await mapper(env)
         results.push(res)
       } catch (e) {
-        console.log(e)
+        console.log('Error in listEnvelopes mapper', e)
```
Comment (mkobetic) [inline src/Client.ts:372]: Should we use `warn` here too?

**Review by elisealix22 — APPROVED** [review]

**Comment by mkobetic** [comment]
Do we already suppress errors decoding conversation messages? The same error will pop up there for the same message.

**Comment by mkobetic** [comment]
Also we need the same handling on the streaming side too.

**Comment by mkobetic** [comment]
> Do we already suppress errors decoding conversation messages? The same error will pop up there for the same message.

I'm thinking V1 here which doesn't apply to lenster, but I think the point still stands.

**Comment by neekolas** [comment]
We already catch all errors in the mapper functions for streams and `listMessages`. This was the one exception, where the decoding happened separately from the mapper.

**Review by mkobetic — APPROVED** [review]

**Pull request merged**